Formula/open-mpi.rb

```diff
@@ -1,10 +1,46 @@
 class OpenMpi < Formula
   desc "High performance message passing library"
   homepage "https://www.open-mpi.org/"
-  url "https://www.open-mpi.org/software/ompi/v3.0/downloads/openmpi-3.0.1.tar.bz2"
-  sha256 "663450d1ee7838b03644507e8a76edfb1fba23e601e9e0b5b2a738e54acd785d"
   revision 1
 
+  stable do
+    # These can be put back up between `homepage` and `revision` once the following patches arrive
+    # upstream in an official Open MPI release:
+    url "https://www.open-mpi.org/software/ompi/v3.0/downloads/openmpi-3.0.1.tar.bz2"
+    sha256 "663450d1ee7838b03644507e8a76edfb1fba23e601e9e0b5b2a738e54acd785d"
+
+    # Fix https://github.com/Homebrew/homebrew-core/issues/26009 and its upstream counterpart
+    # https://github.com/open-mpi/ompi/issues/5000 for the time being.
+
+    # (Remove this patch when either Open MPI v3.0.2 or v3.1.0 is released, as it will be included
+    # in both of those releases.)
+    #
+    # From the result of https://github.com/open-mpi/ompi/pull/5001 as backported to Open MPI v3.0.x
+    # by https://github.com/open-mpi/ompi/pull/5029:
```

Comment:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Do you think this comment is too long?  How would you prefer it be worded?